### PR TITLE
window.$ is undefined

### PR DESCRIPTION
**Pull request by mohandere**
Issue fix - https://github.com/dollarshaveclub/reframe.js/issues/12

### CODE REVIEW

File: src/reframe.js [hunk 1]
```diff
@@ -21,8 +21,8 @@ export default function reframe(target, cName) {
   }
 }
 
-if (window.$) {
```
Comment (yowainwright) [inline src/reframe.js]: What about zepto support? :)

Comment (yowainwright) [inline src/reframe.js]: How about something like:
```js
const plugin = window.$ || window.jQuery || window.zepto;
if (plugin) {
  plugin.fn.extend({
    reframe(cName) {
      return reframe(this, cName);
    },
  });
}
```
...?

Comment (mohandere) [inline src/reframe.js]: @yowainwright zepto js support added. The namespace is `Zepto`.

Comment (mohandere) [inline src/reframe.js]: @yowainwright Is that fine?

Comment (yowainwright) [inline src/reframe.js]: @mohandere I'll be merging something to support your request as soon as possible. Thanks for you note! 🙏

Comment (yowainwright) [inline src/reframe.js]: @mohandere I'm declining this pr. An updated fix for your note will be added soon. In the meantime - just use `reframe(el);` ~Thanks!

**Comment by yowainwright** [comment]
@mohandere Thanks for your feedback & notes. I've added updates to the plugin that will hopefully make **reframe.js** easier for you & others to use. ~Appreciate ya! 🙌

**Comment by mohandere** [comment]
@yowainwright Can you please confirm about  this line - 

`var plugin = window.$ || window.jQuery || window.zepto;`

vs 

`var plugin = window.$ || window.jQuery || window.Zepto;`


i think "Z" should be in Caps.

**Pull request closed without merging**